docs/runbook.md

```diff
@@ -41,15 +41,111 @@
 
```

Comment:
It looks we need a extra explanation how variables works. And then in other examples, we don't need to write any variables.